server/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:alpine

```

Comment:
It is still really useful to have a Docker container for server deployment. Anybody running or working on it locally will of course go native and don't need to install Docker. Docker is just so useful for deployment. If the application crashes it will auto restart etc. Otherwise we would have to add in another dependency for that.